frontend/common/src/helpers/util.ts

```diff
@@ -105,3 +105,7 @@ export const emptyToNull = (s: InputMaybe<string>): string | null =>
  */
 export const emptyToUndefined = (s: InputMaybe<string>): string | undefined =>
   empty(s) || s === "" ? undefined : s;
+
+export function uniqueItems<T>(arr: T[]): T[] {
```

Comment:
Could you write a short description of what the function does along with the `param` and `returns` descriptions, similar to `emptyToUndefined` for future use?

Reply:
Added documentation with [add docs for uniqueItems helper](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3695/commits/c2857b265d636e1f9c16be143a9f7358b40500a6)